sormas-api/src/main/java/de/symeda/sormas/api/utils/jurisdiction/SampleJurisdictionHelper.java

```diff
@@ -21,22 +21,11 @@ public static boolean isInJurisdiction(
 			return false;
 		case NATION:
 			return true;
-		case REGION:
-			return false;
-		case DISTRICT:
-			return false;
-		case COMMUNITY:
-			return false;
-		case HEALTH_FACILITY:
-			return false;
 		case LABORATORY:
-			return sampleJurisdiction.getLabUuid() != null
-				&& DataHelper.equal(sampleJurisdiction.getLabUuid(), userJurisdiction.getHealthFacilityUuid());
+			return sampleJurisdiction.getLabUuid() != null && DataHelper.equal(sampleJurisdiction.getLabUuid(), userJurisdiction.getLabUuid());
 		case EXTERNAL_LABORATORY:
 			return sampleJurisdiction.getOtherLabUuid() != null
-				&& DataHelper.equal(sampleJurisdiction.getOtherLabUuid(), userJurisdiction.getHealthFacilityUuid());
-		case POINT_OF_ENTRY:
-			return false;
+				&& DataHelper.equal(sampleJurisdiction.getOtherLabUuid(), userJurisdiction.getLabUuid());
```

Comment:
Should this really be "otherLabUuid"? The "otherLab" that is still in the database is not actually used in the app, so I think that this field should be removed from the sample jurisdiction as well.

Reply:
removed otherLabUuid from SampleJurisdictionDto and EXTERNAL_LAB check from SampleJurisdictionHelper